sound/hybrid_font.h

```diff
@@ -120,7 +128,37 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (!swing_player_) {
+        swing_player_ = PlayPolyphonic(&swng);
+      }
+      // avoid overlapping swings unless at least a quarter of the swing has played.  Stops double swing events.
+      if ((swing_player_->length() - swing_player_->pos()) / swing_player_->length() <= 0.75) {
+        RefPtr<BufferedWavPlayer> overlap_swing = swing_player_;
+        swing_player_ = PlayPolyphonic(&swng);
```

Comment:
how about move this out and after the if statement, then it can be combined with line 135 I think.

Reply:
I see what you mean.  I've removed the redundant swing_player_ assignment and moved the single call to below the if statement regarding playback position/overlay_swing